content/en/docs/_index.md

```diff
@@ -43,3 +43,21 @@ To build a bootable image is as simple as running `docker build`.
 - Cryptographically verified
 - instant switch from different versions
 - recovery mechanism with `cOS` vanilla images (or bring your own)
+
+## Mission
+
+The cOS-toolkit project is under the Elemental umbrella.
+
+Elemental provides a unique container based approach to define the system lifecycle of an immutable Linux derivative, without any string attached to a specific Linux distribution.
+
+At its heart, Elemental is the abstraction layer between Linux distro management and the specific purpose of the OS.
+
+Elemental empowers anyone to create derivatives from standard OCI images. Frees whoever wants to create a Linux derivative from handling the heavy bits of packaging and managing entire repositories to propagate upgrades, simplifying the entire process by using container images as base for OS.
+At the same time, Elemental provides an highly integrated ecosystem which is designed to be container-first, cloud native, and immutable.
+Anyone can tweak Elemental derivatives from the bottom-up to enable and disable its featureset.
+
+As the Elemental team, the [os2](https://github.com/rancher-sandbox/os2) project is our point of reference.
+
+`os2` is a complete derivative built with Elemental tied with the rancher ecosystem and full cycle node management solution with Kubernetes, which we are supporting directly and indirectly within changes also in the Elemental ecosystem.
```

Comment:
nitpicking: this sentence looks a bit long and harder to understand. I would split it up a little